suite-native/navigation/src/routes.ts

```diff
@@ -6,7 +6,7 @@ export enum RootStackRoutes {
 
 export enum AppTabsRoutes {
     HomeStack = 'HomeStack',
-    Accounts = 'Accounts',
+    AccountStack = 'AccountStack',
```

Comment:
I think this should be plural `AccountsStack`

Reply:
fixed